$OpenBSD: patch-doc_Makefile_am,v 1.1 2019/02/19 08:55:34 rapha Exp $

Upstream commits:
86b14c32a9ab02cab3801382a7ac016ec5047697
39ed30ad834a43cf403f88158c6db7a96f1bed29

Index: doc/Makefile.am
--- doc/Makefile.am.orig
+++ doc/Makefile.am
@@ -29,16 +29,19 @@ man_MANS += \
   unicharset.5
 endif
 
+man_xslt = http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl
+
 EXTRA_DIST = $(man_MANS) Doxyfile
 
 .PHONY: html
 
-html: $(patsubst %,%.html,$(man_MANS))
+html: ${man_MANS:%=%.html}
 
-%: %.asc
-	$(asciidoc) -o $@ $<
+.asc:
+	asciidoc -b docbook -d manpage -o - $< | \
+	xsltproc --nonet $(man_xslt) -
 
-%.html: %.asc
+.asc.html:
 	asciidoc -b html5 -o $@ $<
 
 MAINTAINERCLEANFILES = $(man_MANS) Doxyfile
